File: benchmark/results/ts-invocation-script/run-005/b-baseline/output/src/domain/billing/services/InvoiceRepository.ts
import { Invoice } from "../aggregates/Invoice";

/** InvoiceRepository service. */
export class InvoiceRepository {
  create(input: Partial<Invoice>): Invoice {
    void input;
    throw new Error("not implemented");
  }
  findById(id: string): Invoice | null {
    void id;
    throw new Error("not implemented");
  }
  list(limit: number): Invoice[] {
    void limit;
    throw new Error("not implemented");
  }
  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/billing/services/InvoiceRepository.java
package com.metaengine.demo.billing.services;

import java.util.List;

import com.metaengine.demo.billing.aggregates.Invoice;

/** InvoiceRepository persistence service. */
public class InvoiceRepository {

    /** Persist an Invoice from a partial input. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id (may return null). */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List invoices up to limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
